gitlab/README.md

```diff
@@ -52,15 +55,37 @@ See the [sample gitlab.d/conf.yaml][5] for all available configuration options.
           source: gitlab
     ```
 
-3. [Restart the Agent][9].
+3. [Restart the Agent][5].
+
+#### Containerized
+
+For containerized environments, see the [Autodiscovery Integration Templates][6] for guidance on applying the parameters below.
+
+##### Metric collection
+
+| Parameter            | Value                                                                                                               |
+|----------------------|---------------------------------------------------------------------------------------------------------------------|
+| `<INTEGRATION_NAME>` | `gitlab`                                                                                                            |
+| `<INIT_CONFIG>`      | blank or `{}`                                                                                                       |
+| `<INSTANCE_CONFIG>`  | `{"gitlab_url":"http://%%host%%/", "prometheus_endpoint":"http://<PROMETHEUS_ENDPOINT>:<PROMETHEUS_PORT>/metrics"}` |
```

Comment:
I feel like `%%host%%` shoud go on `prometheus_endpoint` not sure what's the purpose of `gitlab_url` also we might want to replace `<PROMETHEUS_PORT>` with the default port they use to expose metrics